internal/operands/metrics/resources.go

```diff
@@ -76,6 +76,22 @@ func newPrometheusRule(namespace string) *promv1.PrometheusRule {
 								"severity": "Warn",
```

Comment:
@borod108  Its a bug here  "severity": "warning".
Above severity for "SSPDown" and "SSPTemplateValidatorDown" alerts should be "critical" with lower case.

Reply:
OK, will fix in separate PRs for others - is this a convention? because I did see the Alerts firing properly on my env.